Sources/MapboxCoreNavigation/PassiveNavigationFeedbackType.swift

```diff
@@ -49,7 +49,7 @@ public enum PassiveNavigationFeedbackType: FeedbackType {
             return "incorrect_speed_limit"
         case .roadIssue(subtype: .streetPermanentlyBlockedOff):
             return "street_permanently_blocked_off"
-        case .roadIssue(subtype: .streetTemporaryBlockedOff):
+        case .roadIssue(subtype: .streetTemporarilyBlockedOff):
             return "street_temporary_blocked_off"
```

Comment:
Should we also correct the typoe in the raw value, or was this spelling intentional for matching what the backend expects?

Reply:
Thanks for noticing! This constant was in the design doc, and Android merged with the same typo too. Server engineers told me that they don't have a hardcoded list of feedback types, so let's change the raw value too.
I asked Android folks to make the changes too.